myst_nb/static/mystnb.css

```diff
@@ -281,65 +281,72 @@ tbody span.pasted-inline img {
  *
  * It was before in https://github.com/executablebooks/sphinx-book-theme/blob/eb1b6baf098b27605e8f2b7b2979b17ebf1b9540/src/sphinx_book_theme/assets/styles/extensions/_myst-nb.scss
 */
-
-div.cell.tag_output_scroll div.cell_output, div.cell.tag_scroll-output div.cell_output {
-	 max-height: 24em;
-	 overflow-y: auto;
-	 max-width: 100%;
-	 overflow-x: auto;
-}
-
-div.cell.tag_output_scroll div.cell_output::-webkit-scrollbar, div.cell.tag_scroll-output div.cell_output::-webkit-scrollbar {
-	 width: var(--mystnb-scrollbar-width);
-	 height: var(--mystnb-scrollbar-height);
-}
-
-div.cell.tag_output_scroll div.cell_output::-webkit-scrollbar-thumb, div.cell.tag_scroll-output div.cell_output::-webkit-scrollbar-thumb {
-	 background: var(--mystnb-scrollbar-thumb-color);
-	 border-radius: var(--mystnb-scrollbar-thumb-border-radius);
-}
-
-div.cell.tag_output_scroll div.cell_output::-webkit-scrollbar-thumb:hover, div.cell.tag_scroll-output div.cell_output::-webkit-scrollbar-thumb:hover {
-	 background: var(--mystnb-scrollbar-thumb-hover-color);
-}
-
-@media print {
-	 div.cell.tag_output_scroll div.cell_output, div.cell.tag_scroll-output div.cell_output {
-		 max-height: unset;
-		 overflow-y: visible;
-		 max-width: unset;
-		 overflow-x: visible;
-	}
+div.cell:is(
+    .tag_output_scroll,
+    .tag_scroll-output,
+    .config_scroll_outputs
+  )
+  div.cell_output,
+div.cell.tag_scroll-input div.cell_input {
+  max-height: 24em;
+  overflow-y: auto;
+  max-width: 100%;
+  overflow-x: auto;
```

Comment:
Oh! I forgot to mention: this is a little bit tangential to this PR. I realized the style definitions were redundant so I combined input and output styles. And CSS selector was too long so I simplified it with `:is()`